lk2nd/device/dts/msm8909/rules.mk

```
@@ -6,6 +6,7 @@ QCDTBS += \
 	$(LOCAL_DIR)/msm8905-qrd-skub.dtb \
 	$(LOCAL_DIR)/msm8909-1gb-qrd-skuc.dtb \
 	$(LOCAL_DIR)/msm8909-mtp.dtb \
+	$(LOCAL_DIR)/msm8909-qrd-skub.dtb
```

Let's keep trailing nl always escaped so the next person's diff is only one line addition too

```suggestion
	$(LOCAL_DIR)/msm8909-qrd-skub.dtb \
```